roles/galaxy-route/defaults/main.yml

```diff
@@ -29,3 +29,4 @@ ingress_annotations: |
   nginx.ingress.kubernetes.io/proxy-connect-timeout: {{ nginx_proxy_connect_timeout }}
   nginx.ingress.kubernetes.io/proxy-send-timeout: {{ nginx_proxy_send_timeout }}
   haproxy.router.openshift.io/timeout: {{ haproxy_timeout }}
+  route.openshift.io/termination: {{ route_tls_termination_mechanism | lower }}
```

Comment:
ingress_annotations is only used for ingress resources not routes so we should have to add this here

Reply:
I was thinking users could have ingress on OCP, so I followed:
https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/networking/configuring-routes#nw-ingress-creating-a-route-via-an-ingress_route-configuration
But now I see the `route_tls_termination_mechanism` is always associated with `ingress_type | lower == 'route'`